src/index.ts

```diff
@@ -615,8 +617,10 @@ const renderConferencePage = (roomName: string, jwt: string) => {
     }
   });
 
-  document.getElementById("talk")!.style.backgroundColor =
-    options.interfaceConfigOverwrite.DEFAULT_BACKGROUND;
+  let talkEleBgColor = document.getElementById("talk")?.style.backgroundColor;
```

Comment:
Q: `talkEleBgColor` is defined, but never used in this function. it seems to me that lines 620-623 could be removed. Do you agree?

Reply:
I did this to resolve the no-non-null-assertion error from ESLint.
https://palantir.github.io/tslint/rules/no-non-null-assertion/

I could always remove the `talkEleBgColor` variable and simply check if `document.getElementById("talk")?.style.backgroundColor` is non null if that makes it more clear!

Reply:
right. as far as i can tell the variable is never used, so. removing the four lines would be best.

Reply:
There were only two commits I could find using `git log` regarding these lines and both did not tell me much. 

Regarding the intent of the line, `talk` is the id of the `body` element in our index.html file. When `renderConferencePage` is called (by clicking the start call button for example), this line sets the background color of the `body` element to #3B3E4F. 

However, this `body` element is never actually visible in the home page or the call page. I observed that the background color of this element actually changed only by deleting all of the `body` element's children as well as the `head` element through the browser dev tools.

I think that these lines are safe to delete seeing as they serve to make only visual changes that aren't even visible.